.github/PULL_REQUEST_TEMPLATE/tambah_api.md

```diff
@@ -9,16 +9,48 @@ labels: api-baru
 
 **Nama API**:
 **Pengembang**:
-**Tautan Profil Pengembang**:
 **Tautan Dokumentasi**:
-**Status**:
 **Deskripsi**:
-**Autentikasi**:
 
-### Penjelasan API
+## Kategori
 
-Jelaskan secara singkat tentang bagaimana API anda bekerja dan apa saja yang harus dilakukan oleh peserta.
+Silahkan pilih karakteristik yang sesuai dengan API yang Anda tambahkan.
+
+- [ ] Keagamaan / Rohani
+- [ ] Berita / Media
+- [ ] Buku / Literatur
+- [ ] Cuaca
+- [ ] E-Commerce
+- [ ] Finansial / Bank / Kripto
+- [ ] Hiburan / Entertainment / Sosial Media
+- [ ] Jasa Pengiriman
+- [ ] Kesehatan
+- [ ] Lokasi / Maps
+- [ ] Pemerintahan / Government
+- [ ] Pendidikan
+- [ ] Umum / Utilitas
+- [ ] Lainnya
+
+## Tambahan
+
+Silahkan pilih karakteristik yang sesuai dengan API yang Anda tambahkan.
+
+- [ ] No Auth
+- [ ] Authentication Required
+- [ ] Official API
+- [ ] Registration Required
+- [ ] Government API
+- [ ] Third Party Services
+- [ ] Open Data
+- [ ] Dataset
+- [ ] Deprecated
+- [ ] Scraping
+- [ ] Proxy
+- [ ] Self Host
+- [ ] Tools
+- [ ] Others
+- [ ] Paid API
 
 ---
 
-Dengan mengajukan pull request atau issue pada repositori ini, saya menyatakan bahwa tautan RESTful API yang saya sertakan tidak akan mengandung konten NSFW (Not Safe for Work).
+Dengan mengajukan pull request pada repositori ini, saya menyatakan bahwa API yang saya tambahkan adalah API yang legal dan tidak melanggar hak cipta. Saya juga menyatakan bahwa API yang saya tambahkan tidak melanggar hukum dan tidak melanggar privasi pengguna.
```

Comment:
konten NSFW ngga di eksplisit-in aja mas disini?

Reply:
Sepertinya untuk NSFW ini pun saya kurang memahami apakah dijadikan kategori yang terlarang atau tidak di dalam API ini. Memang jika tidak eksplisit diutarakan, sepertinya masih bisa dicover dengan statement tidak melanggar hukum.

Reply:
dari obrolan saya dan kang Fariz, NSFW sudah **tidak diizinkan** disini mas, mengingat generasi-generasi sekarang yang rusak akibat p\*rn\*grafi. jadi kita harap API yang terdaftar di sini hanya API-API yang positif saja.